From d5daa477dcd72938d397aed31295c3867ce5e46b Mon Sep 17 00:00:00 2001
From: Ajay Kumar Gupta <ajay.gupta@ti.com>
Date: Wed, 23 May 2012 15:32:37 +0530
Subject: [PATCH 09/19] usb: musb: ti81xx: print the usbss revision id during
 init

This is to identify the id which can help in debugging.

Signed-off-by: Ravi B <ravibabu@ti.com>
---
 drivers/usb/musb/ti81xx.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/usb/musb/ti81xx.c b/drivers/usb/musb/ti81xx.c
index e3cf38b..f7b2f62 100644
--- a/drivers/usb/musb/ti81xx.c
+++ b/drivers/usb/musb/ti81xx.c
@@ -1069,6 +1069,8 @@ int ti81xx_musb_init(struct musb *musb)
 	if (!rev)
 		return -ENODEV;
 
+	pr_info("MUSB%d controller's USBSS revision = %08x\n", musb->id, rev);
+
 	if (is_host_enabled(musb))
 		setup_timer(&musb->otg_workaround, otg_timer,
 					(unsigned long) musb);
-- 
1.7.7.6

